tests/forms_tests/tests/test_formsets.py

```diff
@@ -461,6 +461,66 @@ class Person(Form):
         self.assertTrue(p.is_valid())
         self.assertEqual(len(p.deleted_forms), 1)
 
+    def test_delete_fields_are_not_added_for_extra_formset_forms(self):
+
```

Comment:
Please try to follow the whitespace style of other tests. We don't need a blank line before the docstring and after every comment in the test.